.vitepress/theme/components/VueJobs.vue

```diff
@@ -29,35 +44,32 @@ onMounted(async () => {
         target="_blank"
       >
         <div class="vj-company-logo">
-          <img :src="job.organization.avatar" alt="" />
+          <img :src="job.organization.avatar"
+            :alt="`${job.organization.name}'s logo for hiring purposes on VueJobs.com.`" />
         </div>
         <div
           style="
             overflow: hidden;
             display: flex;
             flex-direction: column;
             justify-content: center;
-          "
-        >
+          ">
           <div class="vj-job-title">{{ job.title }}</div>
           <div class="vj-job-info">
             {{ job.organization.name }} <span>· </span>
-            <span v-if="['ONLY', 'ALLOWED'].includes(job.remote)"
-              >Remote</span
-            >
-            <span v-else>{{ job.locations[0] }}</span>
+            <span>{{job.remote ? "Remote" : job.locations[0]}}</span>
```

Comment:
I checked on Vuejobs.com that user can only choose between remote or just location so it won't be anything except those three that i'd mentioned at line 19.